spec/index.html

```diff
@@ -819,19 +840,32 @@ <h3>Quoted Triples</h3>
       like "employee38 has a `familyName` of 'Smith'" above;
       rather, it is known as a <a data-cite="RDF12-CONCEPTS#dfn-quoted-triple">quoted triple</a>.</p>
 
+    <p class="ednote">More examples, using identifiers and showing the expansion to Triple Descriptors,
+      or whatever is selected.</p>
+
     <section id="annotation-syntax">
       <h2>Annotation Syntax</h2>
       <p>Turtle also defines an <dfn data-lt="annotation-syntax">annotation syntax</dfn>
         to both quote and assert a <a data-cite="RDF12-CONCEPTS#dfn-rdf-triple">triple</a>,
         which provides a convenient shortcut.
-        An annotation can be used to both assert a triple and have that triple be the
+        An annotation can be used to both assert a triple,
+        via an implicit or explicit identifier,
+        and have that triple be the
         <a data-cite="RDF12-CONCEPTS#dfn-subject">subject</a> of further triples.
```

Comment:
Perhaps a silly question, but can the identifier of an identified triple not also be the _object_ of further triples? If not, why not?

Reply:
Yes, it can be the subject or object of another triple or triple descriptor.

Reply:
```suggestion
        <a data-cite="RDF12-CONCEPTS#dfn-subject">subject</a> and/or
        <a data-cite="RDF12-CONCEPTS#dfn-object">object</a> of 
        further triples and/or <a>triple descriptors</a>.
```

Reply:
Sorry, I neglected what this was associated. The annotation syntax only allows annotating the triple as the subject, not object. Of course, if you use an explicit identifier for the annotation, that can be used as the object of other triples, or a quoted triple consistent with that and using the same identifier could be used as the object of other triples. A quoted triple can be the subject or object of other triples. I believe the identifier of a quoted triple could also be the subject or object of a triple descriptor, but that would be an odd use case. If there is nesting, I would expect only another triple descriptor to be used as the object of a triple descriptor.

Reply:
```suggestion
        <a data-cite="RDF12-CONCEPTS#dfn-subject">subject</a> of further triples.
        If explicitly identified, the same identifier can then be used as the 
        <a data-cite="RDF12-CONCEPTS#dfn-object">object</a> of additional
        triples and/or <a>triple descriptors</a>.
```